### DISCUSSION TITLE
Multiple Apps in App Trigger Section, Separated With Commas

### DISCUSSION THREAD

**Discussion by sapphired18** [xanderfrangos/twinkle-tray#867] [Ideas]
![image](https://github.com/user-attachments/assets/88dc89ba-e1df-462a-aa2d-8e864694432c)
This is useful when many apps need to have same brightness settings, instead of making new profile for each app

**Comment by xanderfrangos** [comment]
@pc-v2 I'm pretty sure I added that in already. It's just not mentioned in the description. 👍
